src/hooks/http.test.js

```diff
@@ -130,6 +130,172 @@ describe('http hook', () => {
     expect(requestData).toEqual({ body: '' });
   });
 
+  test('httpRequestWriteWrapper -> simple flow -> write(str)', dontExitWithoutDone => {
+    const requestData = {
+      body: '',
+    };
+    const wrapper = httpHook.httpRequestWriteWrapper(requestData);
```

Comment:
can you lower this line to where you use it?
It'll help the reader...

Reply:
👍